net/WebSocketHandler.hpp

```diff
@@ -691,6 +691,15 @@ class WebSocketHandler : public ProtocolHandlerInterface
         return sendFrame(socket, data, len, WSFrameMask::Fin | static_cast<unsigned char>(code), flush);
     }
 
+    virtual bool processInputEnabled() const override
+    {
+        std::shared_ptr<StreamSocket> socket = _socket.lock();
+        if (socket)
+            return socket->processInputEnabled();
+
+        return true;
```

Comment:
Enabled even when we have no socket?

Reply:
Sure - not processing input is a sufficiently bad state to be in that we don't really want to go there I think. =)